syntax = "proto3";
package storyboard;




import "next_builder_sdk/model/next_builder/import_storyboard_node.proto";
import "next_builder_sdk/model/next_builder/storyboard_node.proto";

/*
ImportStoryboardNodes请求
*/
message ImportStoryboardNodesRequest {
    /*
    节点列表
    */
    repeated next_builder.ImportStoryboardNode nodes = 1 ;
}

/*
ImportStoryboardNodes返回
*/
message ImportStoryboardNodesResponse {
    /*
    节点列表
    */
    repeated next_builder.StoryboardNode nodes = 1 ;
}

/*
ImportStoryboardNodesApi返回
*/
message ImportStoryboardNodesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    storyboard.ImportStoryboardNodesResponse data = 4 ;
}

